README.md

```diff
@@ -112,20 +112,33 @@ If you have any questions about using this project, Explorer feel free to contac
 To contribute to this project:
 
 * Fork the repo.
-* Submit a Pull Request **with** test coverage.
+* Submit a Pull Request **with** test coverage. 
 * Follow our PR template, which includes the following sections:
 
 ```markdown
-# What's this PR do?
 
-# Where should the reviewer start?
+## What does this PR do? How does it affect users?
 
-# How should this be manually tested?
+## How should this be tested?
+
+Step through the code line by line. Things to keep in mind as you review:
+ - Are there any edge cases not covered by this code?
+ - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?
+
+Fetch the branch and/or deploy to staging to test the following:
+
+- [ ] Does the code compile without warnings (check shell, console)?
+- [ ] Do all tests pass?
+- [ ] Does the UI, pixel by pixel, look exactly as expected (check various screen sizes, including mobile)?
+- [ ] If the feature makes requests from the browser, inspect them in the Web Inspector. Do they look as expected (parameters, headers, etc)?
+- [ ] If the feature sends data to Keen, is the data visible in the project if you run an extraction (include link to collection/query)?
+- [ ] If the feature saves data to a database, can you confirm the data is indeed created in the database?
+
+## Related tickets?
 
-# Screenshots (if appropriate)
 ```
 
-Would an animated GIF be more informative than a screenshot? Then we recommend [Recordit](http://recordit.co/).
+Screenshots encouraged! Would an animated GIF be more informative than a screenshot? Then we recommend [Recordit](http://recordit.co/).
```

Comment:
Totally unnecessary comment, but [Licecap](http://www.cockos.com/licecap/) is also a great recommendation for animated gifs (and there's windows support, for the non-mac users).